app/gui/docs/CONTRIBUTING.md

```diff
@@ -197,6 +197,15 @@ build tool). The most common options are presented below:
   the `entry_point_ide` function, so you have to compile it to test your code in
   the Enso IDE.
 
+### Updating JS dependencies
+
+JS parts of the project are using [`lerna`](https://lerna.js.org/) to manage JS
+dependencies. When you add a new JS dependency or update an old one to the new
+version, `lerna` must be re-run to trigger `npm install` for all projects. The
+easiest way to do so is to remove `dist/init` file before the next build. You
+can also manually call `npm run install` in `app/ide-desktop/` directory with a
+similar result. Our CI does that for every build at the moment.
```

Comment:
would `./run clean --no-rust` do not work here? I'd prefer to stay with `./run` commands if we can. (Im not sure if the flag is `--no-rust` but we have js/rust specific flags there. If the flag is not working, I think we should make it working and prefer it over manual removal of things.

Reply:
Yeah, it would actually work. I'll change it